Finmer.Editor/FormMain.cs

```diff
@@ -282,6 +282,23 @@ private bool SaveAs()
             if (dlgSave.ShowDialog() != DialogResult.OK)
                 return false;
 
+            else 
```

Comment:
No need for this block to be inside an `else`. The other code path - where the user did not click OK on the save dialog - exits out of the function with `return false;`. This is known as an 'early-out'. Since it's impossible for the save dialog result to be anything other than OK here, the `else` isn't necessary.

Reply:
Alright, that one corrected. Neat to know about the early-out.